### PR TITLE
Remove S3 txfrs from w-t jobs. 

### PR DESCRIPTION
**Pull request by doctoryes**
https://openedx.atlassian.net/browse/DENG-707

Add separate S3 txfr jobs downstream from w-t jobs.

### CODE REVIEW

File: dataeng/jobs/analytics/ModelTransfers.groovy [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+                    colorizeOutput('xterm')
+                }
+                publishers common_publishers(allVars)
+                steps {
+                    virtualenv {
+                        pythonName('PYTHON_3.7')
+                        nature("shell")
+                        systemSitePackages(false)
+                        command(
+                            dslFactory.readFileFromWorkspace("dataeng/resources/model_transfers.sh")
```
Comment (brianhw) [inline dataeng/jobs/analytics/ModelTransfers.groovy:52]: this filename doesn't actually match --- the file is called "model-transfers.sh".

**Review by brianhw — COMMENTED** [review]
Looks good overall.   Just an issue with file name.

File: dataeng/jobs/analytics/ModelTransfers.groovy (hunk 1 above)
Comment (doctoryes) [reply dataeng/jobs/analytics/ModelTransfers.groovy]: Thanks - fixed now.

**Review by bmedx — APPROVED** [review]
LGTM

**Review by pwnage101 — APPROVED** [review]

**Review by brianhw — APPROVED** [review]
LGTM.  👍

**Pull request merged**